dist/README.md

```diff
@@ -0,0 +1,29 @@
+---
+layout: page
+title: Testing
+nav_order: 1
+parent: Developer Overview
+---
+# RAPIDS Accelerator for Apache Spark Distribution Packaging
+
+The distribution module creates a jar with support for the Spark versions you need combined into a single jar.
+
+See the [CONTRIBUTING.md](../CONTRIBUTING.md) doc for details on building and profiles available to build an Uber jar.
```

Comment:
Nit: We're not consistent on the capitalization of "uber jar" in this doc.  Capitalizing it makes me think it relates to a particular company, so personally I'd prefer the lower-case form.

Reply:
I'll lower-case them